docs/enterprise.md

```diff
@@ -61,3 +61,192 @@ On top of this, enterprise package of VictoriaMetrics includes the following imp
+Enterprise docker images have `enterprise` suffix in their names. For example, `victoriametrics/victoria-metrics:vX.Y.Z-enteprise`.
+
+In order to run docker image of VictoriaMetrics enterprise component it is required to provide the license key command-line
+flag similar to the one described in the previous section.
+
+For example, the following command runs [VictoriaMetrics single-node](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) docker image with the specified license:
+```console
+docker run --name=victoria-metrics victoriametrics/victoria-metrics:vX.Y.Z -license={VM_KEY_VALUE}
+```
+
+Alternatively, the license can be specified via `-license-file` command-line flag:
+```console
+docker run --name=victoria-metrics -v /vm-license:/vm-license  victoriametrics/victoria-metrics:vX.Y.Z -license-file=/vm-license
+```
+
+### Helm charts
+
+Helm charts for VictoriaMetrics components are available [here](https://github.com/VictoriaMetrics/helm-charts).
+
+In order to run VictoriaMetrics enterprise helm chart it is required to provide the license key via `license` value in `values.yaml` file 
+and adjust the image tag to the enterprise one. 
+
+For example, the following values file for [VictoriaMetrics single-node chart](https://github.com/VictoriaMetrics/helm-charts/tree/master/charts/victoria-metrics-single)
+is used to provide key in plain-text:
+```yaml
+server:
+  image:
+    tag: vX.Y.Z-enterprise 
+
+license:
+  key: {VM_KEY_VALUE}
+```
+
+In order to provide key via existing secret, the following values file is used:
+```yaml
+server:
+  image:
+    tag: vX.Y.Z-enterprise 
+
+license:
+  secret:
+    name: vm-license
+    key: license
+```
+
+Example secret with license key:
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: vm-license
+type: Opaque
+data:
+  license: {BASE64_ENCODED_LICENSE_KEY}
+```
+
+### Kubernetes operator
+
+VictoriaMetrics enterprise components can be deployed via [VictoriaMetrics operator](https://docs.victoriametrics.com/operator/).
+In order to use enterprise components it is required to provide the license key via `license` field and adjust the image tag to the enterprise one.
+
+For example, the following custom resource for [VictoriaMetrics single-node](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html) 
+is used to provide key in plain-text:
+```yaml
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMSingle
+metadata:
+  name: example-vmsingle
+spec:
+  retentionPeriod: "1"
+  license:
+    key: "VM_KEY_VALUE"
+  image:
+    tag: vX.Y.Z-enterprise 
+```
+
+In order to provide key via existing secret, the following custom resource is used:
+```yaml
+apiVersion: operator.victoriametrics.com/v1beta1
+kind: VMSingle
+metadata:
+  name: example-vmsingle
+spec:
+  retentionPeriod: "1"
+  license:
+    keyRef:
+      name: vm-key
+      key: license
+  image:
+    tag: vX.Y.Z-enterprise 
+```
+
+Example secret with license key:
```

Comment:
should we provide example how to create secret having license in file|console? as far we distribute it as just text

Reply:
good point, added